packages/website-frontend/src/app/user/user.component.ts

```diff
@@ -26,6 +26,8 @@ export class UserComponent extends AutoUnsubscribe implements OnInit {
         this.user = user;
       })
     );
+
+    document.addEventListener('sign-out', () => this.logOut())
```

Comment:
This can just be changed to use a `(sign-out)="logOut()"` directly on the `profile-button` template:
`<profile-button (sign-out)="logOut()" name="{{ user.name }}" avatarUrl="{{ user.avatarUrl }}"></profile-button>`